crates/bevy_window/src/lib.rs

```diff
@@ -1,6 +1,12 @@
 #![allow(clippy::type_complexity)]
+#![warn(missing_docs)]
```

Comment:
Should we just `#[deny(missing_docs)]` here from now on?

Reply:
I kept `warn` since that's what seems to be intended to be put for all crates in the long run, according to #3492.

Reply:
Gotcha!